immut/list/moon.pkg.json

```diff
@@ -4,7 +4,8 @@
     "moonbitlang/core/test",
     "moonbitlang/core/array",
     "moonbitlang/core/coverage",
-    "moonbitlang/core/quickcheck"
+    "moonbitlang/core/quickcheck",
```

Comment:
It seems the list itself doesn't depend on QuickCheck, it should be moved to `test-import`

Reply:
@Yoorkin Do you mean this is an existing error?

Reply:
@Yoorkin Unfortunately, this will cause a build error like so:

```rust
      ╭─[[..]/moonbit-core/immut/list/list.mbt:1007:14]
      │
 1007 │ pub impl[X : @quickcheck.Arbitrary] @quickcheck.Arbitrary for T[X] with arbitrary(
      │              ──────────┬──────────  
      │                        ╰──────────── Package "quickcheck" not found in the loaded packages.
──────╯
```

Reply:
Yes. We have two import configuration: `import` for the package itself, and `test-import` for the dependencies used only in black box test.

Reply:
> Yes. We have two import configurations: `import` for the package itself, and `test-import` for the dependencies used only in black box test.

@Yoorkin Thanks. However, I don't think this can be removed that easily, as per my previous comment.